index.bs

```diff
@@ -3820,6 +3820,7 @@ partial interface MLGraphBuilder {
     The <dfn method for=MLGraphBuilder>layerNormalization(|input|, |options|)</dfn> method steps are:
   </summary>
   <div class=algorithm-steps>
+    1. If |options|.{{MLLayerNormalizationOptions/axes}} does not [=map/exist=], set |options|.{{MLLayerNormalizationOptions/axes}} to the [=/list=] « 1, 2, 3 ».
```

Comment:
```suggestion
    1. If |options|.{{MLLayerNormalizationOptions/axes}} does not [=map/exist=], then set |options|.{{MLLayerNormalizationOptions/axes}} to a new [=/list=], either equal to [=the range=] from 1 to |input|'s [=rank-=], exclusive, if |input|'s [=rank=] is greater than 1, or an empty [=/list=] otherwise.
```

Reply:
Does read clearly enough @huningxin and @inexorabletash?